app/models.py

```diff
@@ -2364,3 +2384,18 @@ def serialize(self):
             "updated_at": get_dt_string_or_none(self.updated_at),
             "logged_in_at": get_dt_string_or_none(self.logged_in_at),
         }
+
+
+class LetterAttachment(db.Model):
```

Comment:
is there a reason we have a singular form here? We have a plural for most our tables like that, for example `notifications`, `services`, `templates`.

Reply:
We are actually, sadly, _really_ inconsistent.

For example we also have `organisation`, `domain`, `annual_billing`, `template_folder`.

And to make the inconsistency worse we even sometimes have the model be plural/singular and the table name be opposite, see ApiKey->api_keys. Sad times.

It's obviously an opinionated choice and fairly contentious (https://stackoverflow.com/questions/338156/table-naming-dilemma-singular-vs-plural-names). I guess I settle on the singular side of things here.

Reply:
I see, I think plural makes more sense to me, but I guess it's a matter of preference.